c/jcsi.c

```diff
@@ -25,6 +25,8 @@
 #include "logging.h"
 #include "csi.h"
 #include "jcsi.h"
+#include "zccLogging.h"
```

Comment:
No need to have official message IDs here. We should make the message debug messages.